android/app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,7 @@
   <uses-permission android:name="android.permission.CAMERA"/>
   <uses-permission android:name="android.permission.INTERNET"/>
   <uses-permission android:name="android.permission.READ_EXTERNAL_STORAGE"/>
+  <uses-permission android:name="android.permission.RECORD_AUDIO"/>
```

Comment:
This is not really needed, right? I wonder why it was added. I bet you didn't add it manually.

Reply:
I didn't, indeed.